app/routes/pages/components/PageDetail.js

```diff
@@ -130,14 +130,19 @@ export const MainPageRenderer = ({
   loggedIn: boolean,
 }) => {
   const pageBanner = page.logo || page.picture; //Splittet fra hverandre, var pageBanner = pic || logo
```

Comment:
I can't believe we have a Norwegian comment here :O Did you pass by this without cringing...???.... 😡

Reply:
I def did not see this, or it would have been yeeted!